Library/Formula/libcec.rb

```diff
@@ -11,17 +11,31 @@ class Libcec < Formula
     sha1 "5b04eb4b157c34ab543082a9c9c0868450412c37" => :mountain_lion
   end
 
+  resource "platform" do
+    url "https://github.com/Pulse-Eight/platform/archive/1.0.9.tar.gz"
+    sha256 "276dec2b8653fcf9bf636ac929bb014cc0963ccacd67179705d39301a9533b1e"
+  end
+
   depends_on "autoconf" => :build
   depends_on "automake" => :build
```

Comment:
Are you sure these are still needed? If it's using `cmake` now it seems unlikely. Perhaps `cmake` is the only build-time requirement needed now?